cmake/FindBlas.cmake

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2022- ETH Zurich
+#
+# authors : Mathieu Taillefumier
+
+
+
+if(NOT
+   (CMAKE_C_COMPILER_LOADED
+    OR CMAKE_CXX_COMPILER_LOADED
+    OR CMAKE_Fortran_COMPILER_LOADED))
+  message(FATAL_ERROR "FindBLAS requires Fortran, C, or C++ to be enabled.")
+endif()
+
+set(COSMA_BLAS_VENDOR_LIST
+  "auto"
+  "MKL"
+  "OPENBLAS"
+  "FLEXIBLAS"
+  "ARMPL"
+  "GenericBLAS"
+  "CRAY_LIBSCI"
+  "BLIS"
+  "ATLAS"
+  "NONE")
```

Comment:
I was just asking because I found `NONE` in few places, but maybe those cases are handled separately. `NONE` is also present in 3 places in `src/cosma/CMakeLists.txt`, but maybe that's on purpose.

Reply:
you are right but they have a different meaning and they are not exposed to the user at all. I could use OFF as well. Let me fix this so we have something uniform.

Reply:
Whatever you find better. I was just mentioning, because I wasn't sure if it's on purpose.